src/app/modules/authController.js

```diff
@@ -6,6 +6,22 @@ import jwt from '../helpers/jwt';
 
 class UserController {
   static async signup(req, res) {
+    const checkEmail = req.body.email;
+    const checkUser = req.body.username;
```

Comment:
i've been destructuring separetly because i needed a specifiic element of the request body that i could used in function defined under services and the variables was to avoid duplication but okay